src/utils/__snapshots__/xmltools.test.ts.snap

```diff
@@ -75,6 +75,53 @@ exports[`InterfaceGenerator generates the correct ItemSet code 1`] = `
 "
 `;
 
+exports[`InterfaceGenerator generates the correct OptionSet code 1`] = `
+"export interface optionSet {
+  /**
+   * Multi-selection OptionSet
+   */
+  checkOptionSet: {
+    /**
+     * Selected
+     */
+    _selected: string|Array;
```

Comment:
The Array here should also have the generic type of `string`. And add some spaces on the sides of `|`.

```suggestion
    _selected: string | Array<string>;
```